### DISCUSSION TITLE
How To Contribute

### DISCUSSION THREAD

**Discussion by ujjwaltwitx** [tweepy/tweepy#1607] [Questions] [answered]
I am a begineer in Open Source Contribution and I am not able to understand the project which is of prime importance so that I could contribute to the project. Please someone help me so that I may contribute to the project.

**Comment by Harmon758** [comment, marked as answer]
There are various ways to do so. For example, you can report any issues you encounter.
You'll need some understanding of the library if you want to make code contributions though.
